Execution environment test builds (#69)
ansible-community/antsibull-nox: docs/config-file.md

```toml
[sessions.ansible_lint]
```

## Execution environment check

antsibull-nox allows you to test your collection against an execution environment (EE).

> The `ee-check` session is added with the `[[sessions.ee_check.execution_environments]]` section in `antsibull-nox.toml`.

Reviewer: ```suggestion
The `ee-check` session is added with the `[[sessions.ee_check.execution_environments]]` sections (one for every EE) in `antsibull-nox.toml`.
```

It accepts the following options:

> * `name: str` (Required field): Specifies a unique name for the `ee-check` session.

Reviewer: ```suggestion
* `name: str` (**required**): Specifies a unique name for the `ee-check` session.
```
Same for the other required ones.

> * `description: t.Optional[str]` (default `None`): Specifies a description for the `ee-check` session.

Reviewer: ```suggestion
* `description: str | None` (default `None`): Specifies a description for the `ee-check` session.
```

* `test_playbooks: list[str]` (Required field): Specifies a list of playbooks that test the collection against the EE.
* `version: t.Literal[3]` (default `3`): Configures the schema version for the EE definition.
* `base_image_name: str` (default `"registry.fedoraproject.org/fedora-toolbox:latest"`): Specifies the base image to use when building the EE.

> * `ansible_core_source: t.Literal["package_pip", "package_system"]` (default `"package_pip"`): Configures the source for installing the `ansible-core` package.

Reviewer: ```suggestion
* `ansible_core_source: "package_pip" | "package_system"` (default `"package_pip"`): Configures the source for installing the `ansible-core` package.
```

* `ansible_core_package: str` (default `"ansible-core"`): Specifies the name of the `ansible-core` package.
* `ansible_runner_package: str` (default `"ansible-runner"`): Specifies the name of the `ansible-runner` package.
* `system_packages: list[str]` (default `"[]"`): Specifies a list of system packages to build into the EE.
* `python_packages: list[str]` (default `"[]"`): Specifies a list of Python packages to build into the EE.

> * `python_interpreter_package: t.Optional[str]`(default `None`): Defines the Python system package name for the EE.
> * `python_path: t.Optional[str]`(default `None`): Specifies the path to the Python interpreter.

Reviewer: ```suggestion
* `python_interpreter_package: str | None`(default `None`): Defines the Python system package name for the EE.
* `python_path: str | None`(default `None`): Specifies the path to the Python interpreter.
```
